### PR TITLE
fix(build): Centralized babel config to prevent accidental mismatches

### LINKED ISSUE #292: ReferenceError: exports is not defined
relevant dependencies:

- `polished` version: 1.9.1
- `JSS-in_CSS` library and version: styled-components 2.4.0
- Any relevant JS library and version: react 16.2.0

issue:
updating polished from 1.9.0 to 1.9.1 causes the following error at runtime:
```
Uncaught ReferenceError: exports is not defined
    at Object.eval (polished.es.js?435d:8)
```
I locked all my dependencies to verify that polished 1.9.0 -> 1.9.1 is causing the issue.

this is preventing my app from running at all.

it looks like updates from 1.9.1 added an implicit runtime dependency on CommonJS.

I'll look through some recent diffs to see if I can identify the culprit.

_I love this project! thanks for opening it up to the world!_

### PR DESCRIPTION
**Pull request by Andarist**
Fixes #292 

I think `add-module-exports` plugin is not working at the moment, still investigating this.

EDIT:// 🙈 ^ this was a babel plugin ordering issue, i've swapped transform-commonjs and add-module-exports and it works

### CODE REVIEW

**Comment by Andarist** [comment]
@mxstbr I've seen you are active at the moment, would you be willing to review this and do an emergency bugfix release? `latest` version is not working for majority of users (every user of module-aware bundlers using main entry point - `polished`)

File: .babelrc.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const { BABEL_ENV, NODE_ENV } = process.env
+const cjs = BABEL_ENV === 'cjs' || NODE_ENV === 'test'
+
+module.exports = {
+  presets: [
+    ['env', {
+      modules: false,
```
[the remark this answers is not in the document]
Comment (bhough) [reply .babelrc.js]: If you look at the rollup config, this was already the case, so this will just solve that inconsistency.

**Comment by bhough** [comment]
@Andarist Thanks for jumping on this. CI should cut the release here in a few minutes. Can you verify it is working once that release goes out. I'm traveling so I don't have access to my setup atm.

**Comment by diessica** [comment]
**It wasn't really working** but now it is. Thanks for acting fast on this, I was about to open an issue.

**Comment by mxstbr** [comment]
Oh yeah I forgot that we automatically release polished from CI, that's super neat ✨

**Pull request merged**